doc/sphinxext/numpydoc/tests/test_docscrape.py

```diff
@@ -1,6 +1,7 @@
 # -*- encoding:utf-8 -*-
 from __future__ import division, absolute_import, print_function
 
+import re
```

Comment:
can you move this to pandas/tests/scripts as well? (maybe tricky to get this to import correctly though)

Reply:
I would personally keep this here, it is a vendored package, not our own custom sphinx extension